src/blog/2024/04/how-to-send-and-receive-emails-using-node-red.md

```diff
@@ -0,0 +1,153 @@
+---
+title: How to Send and Receive Emails using Node-RED
+subtitle: Step-by-step Guide for Sending and Receiving Emails using Node-RED
+description: Learn how to send and receive emails using Node-RED, along with best practices for sending email notifications.
+date: 2024-04-03
```

Comment:
```suggestion
date: 2024-04-19
```

Reply:
@Yndira-E Can you merge when the tile is published?

Reply:
Will do!